_posts/2017-06-03-numfocus-grant.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: NumFOCUS small grant for Python 3 support
+---
+<img
+src="http://www.numfocus.org/uploads/6/0/6/9/60696727/1457562110.png"
+title="NumFOCUS Foundation" alt="NumFOCUS Foundation"
+style="float: right; width: 10em;" />
+
+We have generously been awarded a [small development grant][grant-post] by
+NumFOCUS to fully support Python 3. To do this [Richard Gowers][rg]
+and [Tyler Reddy][tr] will be hosted at [Oliver
+Beckstein's][ob] [lab at Arizona State University][beckstlab] in the summer for
+a week of hacking.
+
+MDAnalysis started almost 10 years ago
+when Python was around version 2.4 and interfacing with existing C code was
+mostly done with writing C-wrappers that directly used CPython. This legacy code
+has hampered a speedy full transition to Python 3 and consequently MDAnalysis
+lags behind the rest of the scientific Python community in fully supporting
+Python 3.
+Although about 80% of code passes unit tests in Python 3, we urgently need to
+close the remaining 20% gap in order to support our user base and to safeguard
+the long term viability of the project.
+
+In the meantime we are busy porting our last Python 2.7 only C-extension, the
+DCD Reader and Writer, to Cython. We now have a working cython version that can
+be used without MDAnalysis, similar to our XTC and TRR readers, only cleaning up
```

Comment:
I'd finish the sentence and start a new one at the comma. "Only a clean up of the new Cython / DCD handling code and updated documentation is required." Or something like that.